### PR TITLE
add transactions null check

### PR DESCRIPTION
**Pull request by kwarszawski**
@johanwilfer see if that make sense :)

### CODE REVIEW

File: src/SIE/Data/Verification.php [hunk 1]
```diff
@@ -173,8 +173,12 @@ public function addTransaction(Transaction $transaction)
      * Get all transactions
      * @return Transaction[]
      */
-    public function getTransactions()
+    public function getTransactions(): array
```
Comment (johanwilfer) [inline src/SIE/Data/Verification.php:176]: I think the proper fix is to initialize it in the constructor instead, can you do that and revert these changes?

Well spotted!

Comment (kwarszawski) [reply src/SIE/Data/Verification.php]: I added both property initialization and constructor initialization
This makes it clear that $transactions is intended to be an array, even without looking at the constructor. But if all you need is constructor initialization I'll remove property init ;)
